keras_cv/utils/fill_utils.py

```diff
@@ -80,3 +82,20 @@ def fill_rectangle(images, centers_x, centers_y, widths, heights, fill_values):
 
     images = tf.where(is_rectangle, fill_values, images)
     return images
+
+
+def gather_channels(*matrices, indices=None):
```

Comment:
This one doesn't seem have to anything to do with fill. We might just move it to the loss module for now, until there is any reuse case.

Reply:
Relocated.
Done.